src/modules/fetchlike.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable import/prefer-default-export */
+export const fetchlike = async (key) => fetch(
+  `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${key}/likes`,
+)
+  .then((res) => res.json())
+  .then((output) => output);
```

Comment:
Kindly enable eslint for it is not allowed to disable linters. linters help us write clean code therefore it is against microverse policy to disable linters.